src/reducers/Jobs.js

```diff
@@ -168,12 +174,20 @@ const jobs = (state = {...initialState, ...initialFilters}, action) => {
 
     case JOBS_RETRIEVED: {
       const jobs = addListToStore(state, action.payload);
+      // this is shit
+      const runIds = [];
+      for (const k in state.byId) {
```

Comment:
this kind of for loop here is really bad. first extract it to some function and then try to use `map` instead of a for loop.

Reply:
I tried to use .map but I can't use it on an object like json. I searched on internet how to iter on a json and I find out that I need to use a for loop